pwa/src/templates/sidebar/Sidebar.module.css

```diff
@@ -30,4 +30,5 @@
   max-width: 100%;
   max-height: 100%;
   object-fit: contain;
+  cursor: url(), pointer;
```

Comment:
This line of code is not necessary to the issue but for a different issue.
I think this issue is already merged so I suggest to pull te `refinement-demo-branch` into this branch to avoid merge conflicts